packages/vue/src/components/molecules/SfFilter/_internal/SfFilterItem.vue

```diff
@@ -1,5 +1,5 @@
 <script src="./SfFilterItem.js" />
 <template lang="html" src="./SfFilterItem.html"></template>
 <style lang="scss">
-@import "~@storefrontui/shared/styles/components/SfFilterItem.scss";
+@import "~@storefrontui/shared/styles/components/SfFilter.scss";
```

Comment:
doesn't need to be imported here at all (applies to all child-level components). Styles are not scoped so only parent styles are needed)

Reply:
Oh yeah it's obvious, how can I miss it :laughing: